drivers/modem/modem_iface_uart_interrupt.c

```diff
@@ -198,9 +198,9 @@ int modem_iface_uart_init_dev(struct modem_iface *iface,
 	return 0;
 }
 
-int modem_iface_uart_init(struct modem_iface *iface,
-			  struct modem_iface_uart_data *data,
-			  const struct device *dev)
+int modem_iface_uart_init(struct modem_iface *iface, struct modem_iface_uart_data *data,
+			char *rx_rb_buf, size_t rx_rb_buf_len, const struct device *dev,
+			bool hw_flow_control)
```

Comment:
This is the fifth whitespace inconsistency I've found. I'm going to stop pointing them out at this point.